src/dmd/glue.d

```diff
@@ -76,6 +76,7 @@ __gshared
 {
     elem *eictor;
     Symbol *ictorlocalgot;
+    Symbol* bzeroSymbol;        /// common location for immutable zeros
```

Comment:
Make this a static variable of `getBzeroSymbol`

Reply:
Can't. It has one other use in glue.d

Reply:
cc @Geod24 didn't you recently get rid of a whole bunch of these?

Reply:
Right, but they are better addressed in a later PR I guess.